build.sbt

```diff
@@ -39,7 +39,7 @@ libraryDependencies ++= Seq(
 )
 
 // Wartremover
-wartremoverErrors ++= Warts.unsafe
+wartremoverErrors ++= Warts.unsafe.filterNot(_.clazz == Wart.Throw.clazz)
```

Comment:
I’d rather try to use them all, then be explicit about disabling the ones that cause us trouble.